jina/executors/evaluators/embedding/__init__.py

```diff
@@ -0,0 +1,19 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+
+from .. import BaseEvaluator
+
+
+class BaseEmbeddingEvaluator(BaseEvaluator):
+    """A :class:`BaseEmbeddingEvaluator` evaluates the difference between doc and groundtruth embeddings
```

Comment:
maybe also change to: between actual and desired embeddings

Reply:
done